modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -155,9 +155,16 @@ impl ClientModuleInit for WalletClientInit {
             .0
             .clone()
             .unwrap_or(WalletClientModule::get_rpc_config(args.cfg()));
+
+        // FIXME: reactivate key derivation once we implement recovery
+        let random_root_secret = {
+            let (key, salt): ([u8; 32], [u8; 32]) = thread_rng().gen();
+            DerivableSecret::new_root(&key, &salt)
+        };
+
         Ok(WalletClientModule {
             cfg: args.cfg().clone(),
-            module_root_secret: args.module_root_secret().clone(),
+            module_root_secret: random_root_secret,
```

Comment:
Just to double check - we're fine with this being random *on every start*?

Reply:
The only usage is here:

https://github.com/fedimint/fedimint/blob/d5ade3ee36ba748268686cd4235cb358d3a639b3/modules/fedimint-wallet-client/src/lib.rs#L284-L288

Previously this was always a random key. It being derived helped @douglaz with recovering from the timeout bug, but we could have extracted the key from the state machine log as well.